scripts/clear-audit-boards.py

```diff
@@ -0,0 +1,41 @@
+# pylint: disable=invalid-name
+import sys
+
+from server.models import *  # pylint: disable=wildcard-import
+from server.database import db_session
+
+if __name__ == "__main__":
+    if len(sys.argv) != 2:
+        print("Usage: python -m scripts.clear-audit-boards <jurisdiction_id>")
+        sys.exit(1)
+
+    jurisdiction_id = sys.argv[1]
+    jurisdiction = Jurisdiction.query.get(jurisdiction_id)
+    if not jurisdiction:
+        print("Jurisdiction not found")
+        sys.exit(1)
+
+    print(f"Jurisdiction: {jurisdiction.name}")
```

Comment:
suggest printing the election name & org name, too.

Reply:
good call, added